Makefile

```diff
@@ -214,6 +214,14 @@ image-build-iso: rpm
 iso: image-build-configure image-build-iso
 .PHONY: iso
 
+rpm-builder:
+	podman build -v /etc/pki/entitlement/:/etc/pki/entitlement -t rpm-builder - < ./hack/Containerfile.rpm-builder
+.PHONY: rpm-builder
+
+rpm-in-container:
```

Comment:
Let's simplify the process by having one top-level target? 

We already have 'make rpm' & 'make srpm' - can we add 'make rpm-podman'?